NotesApi/V1/Controllers/BaseController.cs

```diff
@@ -1,38 +1,17 @@
 using Hackney.Core.Middleware;
 using Microsoft.AspNetCore.Mvc;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Serialization;
 using System.Collections.Generic;
 
 namespace NotesApi.V1.Controllers
 {
     public class BaseController : Controller
     {
-        public BaseController()
-        {
-            ConfigureJsonSerializer();
-        }
-
         public string GetCorrelationId()
         {
             if (HttpContext.Request.Headers[HeaderConstants.CorrelationId].Count == 0)
                 throw new KeyNotFoundException("Request is missing a correlationId");
 
             return HttpContext.Request.Headers[HeaderConstants.CorrelationId];
         }
-        public static void ConfigureJsonSerializer()
-        {
-            JsonConvert.DefaultSettings = () =>
-            {
-                var settings = new JsonSerializerSettings();
-                settings.Formatting = Formatting.Indented;
-                settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
-
-                settings.DateTimeZoneHandling = DateTimeZoneHandling.Utc;
-                settings.DateFormatHandling = DateFormatHandling.IsoDateFormat;
-
-                return settings;
-            };
-        }
```

Comment:
How come this is removed?

Reply:
I believe this logic was moved to [Hackney.Core.DynamoDB](https://github.com/LBHackney-IT/lbh-core/blob/release/Hackney.Core/Hackney.Core.DynamoDb/Converters/DynamoDbObjectConverter.cs).